docs/source/ADVANCED_DOCUMENTATION.rst

```diff
@@ -0,0 +1,100 @@
+.. _ADVANCED_DOCUMENTATION:
+
+Advanced Documentation
+======================
+
+Welcome to the advanced documentation for DeepCell Kiosk developers. We will go over cluster customization, accessing cluster logs and metrics, less-common deployment workflows, a few design decisions that may be of interest to other developers, and other topics.
+
+.. contents:: :local:
+
+Preliminaries
+-------------
+
+Kiosk Setup Parameters
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. todo::
+
+    Write a description of the parameters and their defaults
+
+Shell Latency
+^^^^^^^^^^^^^
+
+When testing new features or workflows, DeepCell Kiosk developers will often find themselves using the built-in terminal inside the Kiosk. (Accessible via the Kiosk's main menu as the "Shell" option.) This is a standard ``bash`` shell and should be familiar to most developers. If you are using one of the :ref:`advanced Kiosk deployment workflows <Getting Started:Cloud-Based Jumpbox Workflow>` (which increases shell latency slightly), you should avoid printing unknown and potentially large amounts of text to the screen.
+
+This usually only comes up in the context of logs. To prevent this issue, we recommend the following:
```

Comment:
This should not be a block quote.

Reply:
This doesn't appear as a block quote to me when I look on RTD https://deepcell-kiosk.readthedocs.io/en/mrgn-docs/ADVANCED_DOCUMENTATION.html

Reply:
It is a block quote on github though: https://github.com/vanvalenlab/kiosk/blob/30af2df6c968d06572b6e19b1c9debb48180167e/docs/source/ADVANCED_DOCUMENTATION.rst#id8

Reply:
Actually now that I am reading more RSTs on github, it seems like several things do not render well on github while they do on RTD.  I hope there are options for this, as I would like users to be able to use the github docs as well as RTD.

Reply:
I think I've corrected that specific issue. I'd like to get this pull request in and then look at rst/github compatibility issues in the future.